bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_TTH_TTToAll_HToAATo4B_M-MASS/SUSY_TTH_TTToAll_HToAATo4B_M-MASS_customizecards.dat

```diff
@@ -0,0 +1,2 @@
+set param_card mass 25 125.0
+set param_card mass 36 MASS
```

Comment:
If you are changing the mass, what about the width of the particles. Ok for 25 this might already be close but what about 36?

Reply:
The "a" boson width is always assumed to be 0, or small enough that the b-jet energy uncertainty is much larger than the "a" boson width.

Reply:
What is the default width of "a" in the UFO?

Reply:
Here is the full UFO description of the "a" particle.  It appears the width is zero:

`a = Particle(pdg_code = 22,
             name = 'a',
             antiname = 'a',
             spin = 3,
             color = 1,
             mass = Param.ZERO,
             width = Param.ZERO,
             texname = 'a',
             antitexname = 'a',
             charge = 0,
             GhostNumber = 0,
             Y = 0)`

Reply:
Hi @sihyunjeon ,
The 'a01' width (pdg_code = 36) is listed as:
`width = Param.WA01`
where WA01 has:
`value = 0.000249558656`

Reply:
If you are changing the mass but plan to ignore the width you can just pick a width of 0.1 to make it an explicit "assumption".

Reply:
Hi @agrohsje , is there a strong reason we can't just keep it at 0.00025 GeV?  I ask because many other CMS H --> aa analyses use this same model, and don't set the width explicitly.  I'd rather not deviate from their approach (even if it makes no difference from the physics) if there's not a serious motivation for the new number.

Reply:
Ok